src/nvim/os/fs.c

```diff
@@ -114,17 +114,19 @@ bool os_can_exe(const char_u *name, char_u **abspath)
 static bool is_executable(const char_u *name)
   FUNC_ATTR_NONNULL_ALL
 {
+#ifdef WIN32
+    // Windows does not have an executable bit.
+    // Instead check if the file exists and is readable.
+    return os_file_is_readable((char *) name);
+#else
```

Comment:
Yea. I just realized that after I pushed. :/

Also there appears to be this weird problem with `uv_fs_access` on Windows where if you request anything but `W_OK` it short circuits and returns 0 without actually checking the file attributes which means `os_is_file_readable` always returns true even if the user doesn't have permission to access the given file. :(

Personally I feel that's a bug in the libuv implementation so perhaps a libuv PR is in order?
See: https://github.com/libuv/libuv/blob/df62b54aa2b3d916ca442ee84f0678129a848c4f/src/win/fs.c#L1346